packages/cardpay-sdk/sdk/utils/safe-utils.ts

```diff
@@ -475,3 +476,19 @@ export function gasInToken(estimate: Estimate): BN {
 // The number is based off empirical observation of sentry errors; the baseGas difference is usually 12
 // https://github.com/cardstack/card-protocol-relay-service/blob/master/safe.py#L303-L316
 export const baseGasBuffer: BN = new BN('30');
+
+export async function getTokenBalancesForSafe(
+  provider: JsonRpcProvider,
+  tokenAddresses: string[],
+  safeAddress: string
+) {
+  return await Promise.all(
+    tokenAddresses.map(async (tokenAddress: string) => {
+      let token = new Contract(tokenAddress, ERC20ABI, provider);
```

Comment:
I just realized we already have this in the SDK: https://github.com/cardstack/cardstack/blob/main/packages/cardpay-sdk/sdk/assets.ts#L27

Will adjust this to use the SDK function.